src/window.rs

```diff
@@ -46,7 +47,7 @@ fn main() {
     let mut height: u32 = 50;
     let mut cell_width: u32 = 10;
     let mut chance: u8 = 220;
-    let mut sleep: Option<u64> = None;
+    let mut max_fps: u64 = 10;
```

Comment:
I'd rather call this `fps`, and also make the flag `--fps`.

Reply:
I agree, that's probably a better name.
BTW, I don't really understand the piston event loop, I know it also processes `Input` events but if I exclude them then still for every frame that gets drawn with `--max-fps 1` 5 generations pass. I guess `Loop` events aren't always for drawing, I don't understand all of it but I do know that the `draw2d` function really gets run `max_fps` times a second.